### ISSUE TITLE
Automated builds broken

### ISSUE THREAD

**Issue by TripleSpeeder** [BrightID/BrightID#886]
The last successful build completed on September 27: https://github.com/BrightID/BrightID/actions/runs/1279373961

Since then, all Android builds fail in the build step with this error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:stripDebugDebugSymbols'.
> No toolchains found in the NDK toolchains folder for ABI with prefix: arm-linux-androideabi

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 41s
```

**Comment by TripleSpeeder** [comment]
The github action environment switched to a newer image version. This might be causing the issue, but I'm still investigating.
Last successfull build:
```
  Environment: ubuntu-20.04
  Version: 20210919.1
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210919.1/images/linux/Ubuntu2004-README.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210919.1
```

First failed build:
```
  Environment: ubuntu-20.04
  Version: 20210929.1
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210929.1/images/linux/Ubuntu2004-README.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210929.1
```

The older image has 2 NDKs installed:
```
21.4.7075529
22.1.7171670
```
The newer image has 3 NDKS:
```
21.4.7075529 (default)
22.1.7171670
23.0.7599858
```

But both images have the same ANDROID_NDK_HOME set:
`ANDROID_NDK_HOME	/usr/local/lib/android/sdk/ndk-bundle -> /usr/local/lib/android/sdk/ndk/21.4.7075529`
So in theory this should not have an influence...

**Comment by TripleSpeeder** [comment]
Root problem:
- We are using Android Gradle Plugin 3.5.4. This version uses by default the latest installed NDK version
- The newer github action environment image has added NDK version 23.0.7599858. This version is not compatible with Gradle 3.5.4.

There are three possible fixes:
1. Update to AGP 3.6.x. Starting with 3.6 AGP has a built-in default NDK version. See https://developer.android.com/studio/releases/gradle-plugin#default-ndk-version. This requires to manually add the default NDK to the github install image, as it is not included by default.
2. Manually specify the NDK version to use (set it to a version available in the github image)
3. Update to AGP 4.1.x. Starting with 4.1 AGP will automatically install the default NDK version if it is not availble

**Comment by TripleSpeeder** [comment]
Fixed by manually setting NDK version with #888
